Cargo.toml

```diff
@@ -53,6 +52,8 @@ pasta-msm = { version = "0.1.0", package = "lurk-pasta-msm" }
 proptest = "1.1.0"
 proptest-derive = "0.3.0"
 rand = "0.8.5"
+rustyline = { version = "11.0", features = ["with-file-history"], default-features = false }
```

Comment:
I’m not sure this dependency will work as an override of the default dependency. Would it make sense to put the non-features rustyline in the section for `target_arch = “wasm32”`?